src/routes/+page.svelte

```diff
@@ -9,49 +9,59 @@
   } from '$lib/stores';
   import Connect from '$components/Connect.svelte';
   import Capacity from '$components/Capacity.svelte';
-  import Intro from '$components/Intro.svelte';
   import Provider from '$components/Provider.svelte';
   import KeySelection from '$components/KeySelection.svelte';
   import ProviderActions from '$components/ProviderActions.svelte';
-  import ConnectionStatus from '$components/ConnectionStatus.svelte';
+  import ChainStatus from '$components/ChainStatus.svelte';
+  import logo from '$lib/assets/logo.png';
+  import topright from '$lib/assets/top-right-bars.png';
+  import bottomleft from '$lib/assets/bottom-left-bars.png';
+  import type {ChainInfo, MsaInfo} from "$lib/storeTypes";
 
   let token = '';
   let blockNumber = 0n;
+  let epochNumber = 0n;
   let connected = false;
   let validAccounts = {};
-  let showDashboard = false;
 
+  // TODO: put all this in chainInfo and update how it's stored.
   storeBlockNumber.subscribe((val) => (blockNumber = val));
   storeToken.subscribe((val) => (token = val));
   storeConnected.subscribe((val) => (connected = val));
   storeValidAccounts.subscribe((val) => (validAccounts = val));
+  storeChainInfo.subscribe((info: ChainInfo) => (epochNumber = info.epochNumber));
 
   const onChangeTxnSigningAddress = (evt: Event) => {
     let option = evt.target as HTMLOptionElement;
-    storeMsaInfo.set({ isProvider: false, msaId: 0, providerName: '' });
+    storeMsaInfo.set({isProvider: false, msaId: 0, providerName: ''});
     transactionSigningAddress.set(option.value);
   };
 </script>
+<img alt="decoration-top-left" src={topright} class="mt-0 absolute top right-8 -z-40"/>
 
-<h1 class="text-3xl font-bold">Welcome to Provider Dashboard</h1>
-<div id="status-bar" class="flex pr-8 justify-start pt-6 h-52">
-  <ConnectionStatus {blockNumber} {connected} {token} />
-  <Provider />
-  <Capacity bind:token />
+<div class="bg-header mt-8 h-24">
+  <img alt="The project logo" src={logo} class=""/>
+  <p class="text-aqua text-lg font-semibold tracking-wider pl-3">Provider Dashboard</p>
 </div>
-<div id="main-actions" class:hidden={!showDashboard} class="mt-8 text-white">
+
+<ChainStatus {blockNumber} {connected} {token} {epochNumber}/>
+<div class="flex justify-center">
+  <Provider/>
+  <Capacity bind:token/>
+</div>
+<div class="mt-8 text-white">
   <form id="setupForm">
-    <Connect />
-    <div class={connected ? '' : 'hidden'}>
+    <Connect/>
+    <div class:hidden={!connected} class="mt-8">
       <KeySelection
         component="TransactionSigningKey"
-        selectLabel="Choose a Transaction Signing Address"
+        selectLabel="Choose a Wallet Address"
```

Comment:
In the e2e test this also needs to update: `const signer = screen.getByLabelText('Choose a Transaction Signing Address');`